src/utils.ts

```diff
@@ -2,6 +2,50 @@ import * as actions_exec from "@actions/exec";
 import * as core from "@actions/core";
 import * as im from "@actions/exec/lib/interfaces";
 
+// Enum of Gazebo distributions
+enum gazeboDistro {
```

Comment:
The really awesome thing here would to use https://raw.githubusercontent.com/gazebo-tooling/release-tools/master/jenkins-scripts/dsl/gz-collections.yaml to parse the collections from the YAML and the grab the ubuntu distributions from ci_configs or packaging_configs. We can leave it for a new PR or create an issue by now.

Reply:
Sounds like a great idea. I will create a separate issue for this